packages/config/src/config.js

```diff
@@ -25,34 +26,39 @@ module.exports = ({
     routesPath,
     appUrl,
     exactUrl,
-    disableFallback
+    disableFallback,
+    isProd
 } = {}) => {
     const filenameMask = `js/[name]${useFileHash ? '.[chunkhash]' : ''}.js`;
     return {
-        mode: mode || (process.env.NODE_ENV === 'production' ? 'production' : 'development'),
+        mode: mode || (isProd ? 'production' : 'development'),
         devtool: false,
         optimization: {
-            minimize: (process.env.NODE_ENV || mode) === 'production',
+            runtimeChunk: true,
```

Comment:
@redallen have tried this with federated modules? I know that using option `single` will rip out the webpack init code from the federated module entry point so it's impossible to use in a different container. I am not sure about `multiple` though.

EDIT: It seems to have broken the `ChunkMapper` used for generating the `fed-mods.json`. 
```
TypeError: Cannot read property 'modules' of undefined
    at /home/martin/insights/insights-inventory-frontend/node_modules/@redhat-cloud-services/frontend-components-config/chunk-mapper.js:28:69

```

When added a condition to leave the function if the module is not found, no output was generated.

Reply:
Removed.